File: Java/tuition.java
//TUITION

import java.util.*;

public class Tuition {
    static Scanner scanner = new Scanner(System.in);

    public static void main(String[] args) {
        int modeOfPayment;
        float tuitionFee;
        float fullPayment;
        float installmentA;
        float installmentB;

        System.out.println("Tuition");
        System.out.print("Enter tuition: ");
        tuitionFee = scanner.nextFloat();

        if (tuitionFee < 5000) {
            System.out.println("Input error, amount less than 5,000 pesos is not allowed.");
        } else {
            System.out.println("Mode of payment");
            System.out.println("[1] Full payment - 20% Discount");
            System.out.println("[2] Installment A - 5% Interest");
            System.out.println("[3] Installment B - 10% Interest");
            System.out.print("Enter mode of payment: ");
            modeOfPayment = scanner.nextInt();

            switch (modeOfPayment) {
                case 1:
                    fullPayment = tuitionFee - (tuitionFee * 0.20f);
                    System.out.println("The tuition to pay is " + String.format("%.2f", fullPayment) + " pesos.");
                    break;
                case 2:
                    installmentA = tuitionFee * 1.05f;
                    System.out.println("The tuition to pay is " + String.format("%.2f", installmentA) + " pesos.");
                    break;
                case 3:
                    installmentB = tuitionFee * 1.10f;
                    System.out.println("The tuition to pay is " + String.format("%.2f", installmentB) + " pesos.");
                    break;
                default:
                    System.out.println("Input error, please input from 1 - 3 only.");
            }
        }
    }
}

File: Python/tuition.py
print("Tuition")
tuition_fee = float(input("Enter tuition: "))

if tuition_fee < 5000:
    print("Input error, amount less than 5,000 pesos is not allowed.")
else:
    print("Mode of payment")
    print("[1] Full payment - 20% Discount")
    print("[2] Installment A - 5% Interest")
    print("[3] Installment B - 10% Interest")
    mode_of_payment = int(input("Enter mode of payment: "))

    if mode_of_payment == 1:
        full_payment = tuition_fee - (tuition_fee * 0.20)
        print("The tuition to pay is " + f"{full_payment:.2f}" + " pesos.")
    elif mode_of_payment == 2:
        installment_a = tuition_fee * 1.05
        print("The tuition to pay is " + f"{installment_a:.2f}" + " pesos.")
    elif mode_of_payment == 3:
        installment_b = tuition_fee * 1.10
        print("The tuition to pay is " + f"{installment_b:.2f}" + " pesos.")
    else:
        print("Input error, please input from 1 - 3 only.")
